src/Form/FieldAbstract/BaseField.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+namespace Kirby\Form\FieldAbstract;
```

Comment:
`Question:` Do we need this extra namespace? Couldn't this live also in `Kirby\Form\Field`?